src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -83,7 +116,7 @@ public String beforeOrAfter(LocalDate someDate) {
      * @return LocalDateTime
      */
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        return ZonedDateTime.parse(dateInString + "[" + zone + "]").toLocalDateTime();
```

Comment:
let's try to be less straightforward and solve this with the help of Instant class

Reply:
done